<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form_1</class>
 <widget class="QWidget" name="Form_1">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>395</width>
    <height>230</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>395</width>
    <height>230</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>395</width>
    <height>230</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Nagrywanie</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff/>
   </iconset>
  </property>
  <widget class="QLineEdit" name="fnameE">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>70</y>
     <width>381</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>50</y>
     <width>151</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>Podaj nazwę pliku:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="recB">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>190</y>
     <width>181</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Nagrywaj</string>
   </property>
   <property name="iconSize">
    <size>
     <width>24</width>
     <height>24</height>
    </size>
   </property>
   <property name="shortcut">
    <string>X</string>
   </property>
  </widget>
  <widget class="QPushButton" name="stopB">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>190</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Stop</string>
   </property>
   <property name="iconSize">
    <size>
     <width>24</width>
     <height>24</height>
    </size>
   </property>
   <property name="shortcut">
    <string>V</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>130</y>
     <width>91</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>Samplerate:</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="rateB">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>150</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="minimum">
    <number>4000</number>
   </property>
   <property name="maximum">
    <number>48000</number>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>130</y>
     <width>81</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>Ilość bitów:</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="bitsB">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>150</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="minimum">
    <number>8</number>
   </property>
   <property name="maximum">
    <number>32</number>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>130</y>
     <width>101</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>Ilość kanałów:</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="chnB">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>150</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="minimum">
    <number>1</number>
   </property>
   <property name="maximum">
    <number>2</number>
   </property>
  </widget>
  <widget class="QLabel" name="txtL">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>371</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="savB">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>30</y>
     <width>121</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Zapis do pliku</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pauseB">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>190</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Pauza</string>
   </property>
   <property name="iconSize">
    <size>
     <width>24</width>
     <height>24</height>
    </size>
   </property>
   <property name="shortcut">
    <string>C</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="sndB">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>110</y>
     <width>151</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Odtwarzaj dźwięk</string>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>savB</tabstop>
  <tabstop>fnameE</tabstop>
  <tabstop>sndB</tabstop>
  <tabstop>rateB</tabstop>
  <tabstop>bitsB</tabstop>
  <tabstop>chnB</tabstop>
  <tabstop>recB</tabstop>
  <tabstop>pauseB</tabstop>
  <tabstop>stopB</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
